thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -154,7 +162,7 @@ body {
 .te-label {
   font-weight: normal;
   color:  app-shade(black, 9);
-  margin-bottom: 12px;
+  margin: 15px 0 12px 0;
```

Comment:
Since right and left margins are both 0, you can write this as 3 values as such: 
`margin: 15px 0 12px`. Documentation: https://www.w3schools.com/css/css_margin.asp

Reply:
thanks. see comment below - this change was removed.